Challenge-4/Bot_2/src/agent.spec.ts

```diff
@@ -2,81 +2,103 @@ import {
   MockEthersProvider,
   TestTransactionEvent,
 } from "forta-agent-tools/lib/test";
+
 import {
   ethers,
   Finding,
   FindingSeverity,
   FindingType,
   HandleTransaction,
 } from "forta-agent";
+
 import { provideInitialize, provideHandleTransaction } from "./agent";
 import { createAddress } from "forta-agent-tools";
 import Helper from "./helper";
-import { getAlerts } from "forta-agent";
-
-jest.mock("forta-agent", () => ({
-  ...jest.requireActual("forta-agent"),
-  getAlerts: jest.fn(),
-}));
+import { NetworkManager } from "forta-agent-tools";
+import { NetworkData } from "./types";
 
 describe("Compound test suite for lending and borrowing", () => {
   let mockProvider = new MockEthersProvider();
   let handleTransaction: HandleTransaction;
   let txEvent: TestTransactionEvent;
   let initialize: any;
   let helper: Helper;
-  let usdcTokenAddress = "0xc3d688B66703497DAA19211EEdff47f25384cdc3";
-  let mockConfiguratorProxy = "0x316f9708bb98af7da9c68c1c3b5e79039cd336e3";
+
+  let mockEthUsdcTokenAddress = createAddress("0x1423"); 
+  let mockEthConfiguratorProxy = createAddress("0x1524");
+  let mockPolygonmockEthUsdcTokenAddress = createAddress("0x2938");
+  let mockPolygonConfiguratorProxy = createAddress("0x9182");
+
+  let setupMockProvider: any;
```

Comment:
In general, we really try to not declare variables to be of type `any`, since that defeats the purpose of TS. Since this will have the value of a function, look up how to set a type to match a function. This will help you in the long term. 😉